code/src/Components/InputToDo.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { todoitem } from './Reducers/todoitem';
+
+export const InputToDo = () => {
+  const [input, setInput] = useState("");
+  const dispatch = useDispatch();
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    dispatch(todoitem.actions.addItem(input))
+    setInput('')
+  }
+
+  //Add todoitems by mapping the items in the todolist in the store
+  return (
+    <form onSubmit={handleSubmit}>
+      <button
+        type="submit">
+        +
+      </button>
+      <input
+      type="text"
+        placeholder="Add Task"
+        value={input}
+        onChange={(event) => setInput(event.target.value)}
+      >
+      </input>
```

Comment:
Since the input element doesn't have any children, you don't need opening and closing tags, you can just make it self closing:
```
<input
  type="text"
  placeholder="Add Task"
  value={input}
  onChange={(event) => setInput(event.target.value)}
/>
```